src/Parser/VengeanceDemonHunter/CombatLogParser.js

```diff
@@ -122,11 +131,20 @@ class CombatLogParser extends MainCombatLogParser {
       });
     }
 
+    if (spiritBombUptimePercentage < 1.0 && this.selectedCombatant.hasTalent(SPELLS.SPIRIT_BOMB_TALENT.id)) {
```

Comment:
I'll do it on the next PR (I promise :smile:). But that one suggestion needs an correction on the live for a better analysis.
From the next PR and later on I'll separate the new things and refactor the code.